docs/adoc/technicalGuideJS/ScoutJS.adoc

```diff
@@ -1288,10 +1288,66 @@ Output:
 
   <scout-version data-value="16.1.0.002"></scout-version>
 
+== TypeScript Support
+
+Eclipse Scout is written in TypeScript and therefore comes with a fully typed TypeScript API.
+
+This gives you many benefits:
+
+* *More efficient development* +
+Types allows your IDE to better assist with code completion during development and only suggests methods and properties for an object that actually exist. +
++
+.Better Code Completion
+image::{imgsdir}/scoutjs_code_completion.png[]
+* *Easier entry* +
+With better code completion and clear method signatures, a not-so-experienced JavaScript developer can more easily get up to speed and understand the relationships in the code.
+* *More security for upgrades* +
+If a used API has changed, the code turns red and the build fails.
+* *More security during development* +
+Incorrect use of an API or code in general is detected at an early stage. The developer gets more security that his code will work. +
+
+In addition to the TypeScript API, Scout also tries to make it as easy as possible for you so you can write TypeScript yourself in your project.
+This includes:
+
+* An enhanced build stack that can transpile TypeScript code to JavaScript.
+* An improved Scout SDK so IntelliJ can handle Scout code written in TypeScript.
+* Predefined TypeScript configuration and ESLint rules for TypeScript.
+
+=== Project Setup for TypeScript
+
+To use TypeScript in your Project, you need to create a `tsconfig.json` that extends from the Scout `tsconfig.json`.
+Follow the instructions at https://www.npmjs.com/package/@eclipse-scout/tsconfig[@eclipse-scout/tsconfig] to create one.
+
+Now, just create a TypeScript file (a new file ending with `.ts`) and start writing TypeScript!
+
+The build automatically processes these files, so you don't have to adjust the build unless you are writing a JavaScript library.
+If that is the case, see <<Authoring Libraries>>.
+
+To enable code linting for TypeScript files just ensure your ESLint configuration is correctly setup as described in <<ESLint>>.
+The Scout default configuration can handle JavaScript as well as TypeScript code, so there are no separate configurations needed.
```

Comment:
There is no plural for configuration?
E.g. use "... so there is no separate configuration needed."